test/functional/interface_zmq_dash.py

```diff
@@ -89,17 +89,17 @@ def run_test(self):
             self.zmq_context.destroy(linger=None)
 
     def subscribe(self, publishers):
-        # Subscribe a list of ZMQPublisher
+        # Subscribe to a list of ZMQPublishers
```

Comment:
It's an enum `class ZMQPublisher(Enum)` thats why i didn't do a plural there i guess haha.. Probably ok but in case you still want to change it, maybe `Subscribe to a list of ZMQPublisher entries` or so would make more sense..

Reply:
:man_shrugging: